From 93bc3d440c937486a1885050818431746b171cea Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Wed, 10 Feb 2010 17:20:49 +0100
Subject: [PATCH 14/23] make nfs4 work

---
 modules.d/95nfs/install |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/modules.d/95nfs/install b/modules.d/95nfs/install
index 2b2d7cf..ea2d432 100755
--- a/modules.d/95nfs/install
+++ b/modules.d/95nfs/install
@@ -36,6 +36,7 @@ mkdir -p "$initdir/var/lib/nfs/statd/sm"
 
 # Rather than copy the passwd file in, just set a user for rpcbind
 # We'll save the state and restart the daemon from the root anyway
+egrep '^root:' "$initdir/etc/passwd" || echo  'root:x:0:0::/:/bin/sh' >> "$initdir/etc/passwd"
 egrep '^nobody:' /etc/passwd >> "$initdir/etc/passwd"
 egrep '^nfsnobody:' /etc/passwd >> "$initdir/etc/passwd"
 egrep '^rpc:' /etc/passwd >> "$initdir/etc/passwd"
-- 
1.6.6

