From a56e93c10cae00f453111291a907d03b16dca5ef Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Thu, 4 Mar 2010 13:47:40 +0100
Subject: [PATCH 37/64] kernel-modules: add keyboard kernel modules

---
 modules.d/90kernel-modules/installkernel |    4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/modules.d/90kernel-modules/installkernel b/modules.d/90kernel-modules/installkernel
index b966370..0fbafe7 100755
--- a/modules.d/90kernel-modules/installkernel
+++ b/modules.d/90kernel-modules/installkernel
@@ -7,6 +7,10 @@ if [[ -z $drivers ]]; then
     }
     instmods pcmcia sd_mod scsi_dh scsi_dh_rdac scsi_dh_emc
     instmods firewire-ohci
+
+    # install keyboard support
+    instmods atkbd i8042 usbhid hid-apple ehci-hcd ohci-hcd uhci-hcd
+
     instmods "=drivers/pcmcia" =ide "=drivers/usb/storage"
     instmods $(filter_kernel_modules block_module_test) 
     # if not on hostonly mode, install all known filesystems if the required list is not set via the filesystems variable
-- 
1.6.6

