From ed8c7bf5c28ebe11f878443c2b21fd2695181ba0 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Wed, 26 May 2010 17:43:18 +0200
Subject: [PATCH 132/133] dracut-functions: beautified warnings

---
 dracut-functions |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/dracut-functions b/dracut-functions
index fce5abb..a76cc22 100755
--- a/dracut-functions
+++ b/dracut-functions
@@ -387,7 +387,7 @@ should_source_module() {
     "$1/check" $hostonly || return 1
     for dep in $("$1/check" -d); do
 	check_module_deps "$dep" && continue
-	dwarning "Cannot load $mod, dependencies failed."
+	dwarning "Cannot load dracut module \"$mod\", dependencies failed."
 	return 1
     done
 }
@@ -435,7 +435,7 @@ install_kmod_with_fw() {
 	    fi
 	done
 	if [[ $found != yes ]]; then
-	    dinfo "Possible missing firmware ${fw} for module ${mod}.ko"
+	    dinfo "Possible missing firmware \"${fw}\" for kernel module \"${mod}.ko\""
 	fi
     done
 }
-- 
1.7.0.1

