From be0921a413e987f9b213727dea7f198ffa4636ac Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Amadeusz=20=C5=BBo=C5=82nowski?= <aidecoe@aidecoe.name>
Date: Wed, 9 Jun 2010 10:40:31 +0200
Subject: [PATCH 140/148] 90crypt: 'crypto_LUKS' identifier corrected

---
 modules.d/90crypt/check |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/modules.d/90crypt/check b/modules.d/90crypt/check
index 07c53f9..852695b 100755
--- a/modules.d/90crypt/check
+++ b/modules.d/90crypt/check
@@ -19,7 +19,7 @@ is_crypt() { [[ $(get_fs_type /dev/block/$1) = crypto_LUKS ]]; }
 	check_block_and_slaves is_crypt "$rootdev" || exit 1
     else
 	# root is not on a block device, use the shotgun approach
-	blkid | grep -q crypt_LUKS || exit 1
+	blkid | grep -q crypto\?_LUKS || exit 1
     fi
 }
 
-- 
1.7.0.1

