From 08e10f05f61a1d0d4ea42ee265ad843db670c81e Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Wed, 9 Jun 2010 11:56:06 +0200
Subject: [PATCH 142/148] plymouth/cryptroot-ask.sh: beautify password prompt

---
 modules.d/50plymouth/cryptroot-ask.sh |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/modules.d/50plymouth/cryptroot-ask.sh b/modules.d/50plymouth/cryptroot-ask.sh
index 7dee9a1..3897b33 100755
--- a/modules.d/50plymouth/cryptroot-ask.sh
+++ b/modules.d/50plymouth/cryptroot-ask.sh
@@ -68,13 +68,13 @@ unset LUKS luks luuid
 if [ $ask -gt 0 ]; then
     info "luksOpen $device $luksname"
     if [ $num -eq 1 ]; then
-         prompt="Password:"
+         prompt="Password for filesystem"
     else
          prompt="Password [$device ($luksname)]:" 
          if [ ${#luksname} -gt 8 ]; then
 	     sluksname=${sluksname##luks-}
              sluksname=${luksname%%${luksname##????????}}
-             prompt="Password [$device ($sluksname...)]:"
+             prompt="Password for $device ($sluksname...)"
          fi
     fi
     # flock against other interactive activities
-- 
1.7.0.1

