diff -up dracut-004/mkinitrd-dracut.sh.mki dracut-004/mkinitrd-dracut.sh
--- dracut-004/mkinitrd-dracut.sh.mki	2010-08-19 18:11:09.000000000 +0200
+++ dracut-004/mkinitrd-dracut.sh	2010-08-19 20:27:34.000000000 +0200
@@ -142,9 +145,9 @@ if [ -n "$img_vers" ]; then
 fi
 
 if [ -n "$basicmodules" ]; then
-	dracut -H $dracut_args --add-drivers "$basicmodules" "$target" "$kernel"
+	dracut $dracut_args --add-drivers "$basicmodules" "$target" "$kernel"
 else
-	dracut -H $dracut_args "$target" "$kernel"
+	dracut $dracut_args "$target" "$kernel"
 fi
 
 # vim:ts=8:sw=4:sts=4:et
